DisguiseUnityRenderStream/Runtime/TemporaryTexture2DManager.cs

```diff
@@ -112,6 +112,19 @@ public static TemporaryTexture2DManager Instance
             PlayerLoopExtensions.RegisterUpdate<PostLateUpdate.FinishFrameRendering, FinishFrameRendering>(OnFinishFrameRendering);
         }
 
+        public override void Dispose()
+        {
+            foreach (var (_, item) in m_Items)
```

Comment:
Curious about why we need this. Doesn't the runtime clean up the textures when the game quits, or when entering play mode (domain reload).

Reply:
I have not encountered a need for this in any other projects I've seen.